fancontrol.py

```diff
@@ -3,23 +3,19 @@
 import subprocess
 import time
 
-from gpiozero import OutputDevice
+from gpiozero import PWMOutputDevice
 
-
-ON_THRESHOLD = 65  # (degrees Celsius) Fan kicks on at this temperature.
-OFF_THRESHOLD = 55  # (degress Celsius) Fan shuts off at this temperature.
-SLEEP_INTERVAL = 5  # (seconds) How often we check the core temperature.
-GPIO_PIN = 17  # Which GPIO pin you're using to control the fan.
+SLEEP_INTERVAL = 3  # (seconds) How often we check the core temperature.
+GPIO_PIN = 18  # Which GPIO pin you're using to control the fan.
```

Comment:
Does the pin need to change here?

Reply:
I believe it does as not all GPIO pins are PWM enabled.

Reply:
Can you tell me how is its possible to enable multiple GPIO pins using PWM, I am thinking of using one internal PWM controlled fan for the Rpi (5V one) and one for the external 12V Fan which will be temperature based PWM controlled too. so in this case I need two GPIO pins to have PWM functionality. Please guide me in this regard as I am newbie thanks
> I believe it does as not all GPIO pins are PWM enabled.

Reply:
The RPI manual will tell you which of the pins are PWM enabled. They aren't all but there are a few

Reply:
**thanks I will find them but apart from that the idea which I have presented up , does that sound possible on a RPi ? I mean in this case two scripts will be running in the background and will execute after each bootup**